src/main/java/com/barterbay/app/servcie/GoodService.java

```diff
@@ -100,9 +102,9 @@ public void saveGood(CreateGoodDTO goodDTO) {
   private void uploadPhotosToS3(CreateGoodDTO goodDTO, Good good) {
     if (goodDTO.getPhotos() != null) {
       final var photoUrls = storageProxyService.uploadMultipleFiles(
-        goodDTO.getUserOwnerId(), good.getId(),
-        goodDTO.getPhotos(), amazonS3, bucketName
-      ).stream()
+          goodDTO.getUserOwnerId(), good.getId(),
+          goodDTO.getPhotos(), amazonS3, bucketName
+        ).stream()
```

Comment:
I think the formatting is not good here)
Try Ctrl+Alt+L

Reply:
No, that's correct)